samples/matter/weather_station/README.rst

```diff
@@ -0,0 +1,154 @@
+.. _matter_weather_station_sample:
+
+Matter: Weather station
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This weather station sample demonstrates the usage of the :ref:`Matter <ug_matter>` (formerly Project Connected Home over IP, Project CHIP) application layer to build a weather station device that lets you remotely gather different kind of data, such as like temperature, air pressure, and relative humidity, using the device sensors.
+This device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power, 802.15.4 Thread network.
+You can use this sample as a reference for creating your own application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: thingy53_nrf5340
+
+To program and gather logs from the Thingy:53 device, you need the external J-Link programmer.
+If you own an nRF5340 DK that has an on-board J-Link programmer, you can also use it for this purpose.
+
+For remote testing scenarios, if you want to commission the lock device and :ref:`control it remotely <matter_lock_sample_network_mode>` through a Thread network, you also need the following:
+
+* A Matter controller device :ref:`configured on PC or smartphone <ug_matter_configuring>`.
+
+.. note::
+    |matter_gn_required_note|
+
+Overview
+********
+
+The sample uses buttons for changing the lock and device states, and LEDs to show the state of these changes.
```

Comment:
There's just one button on Thingy:53. It's used for opening pairing window (if pressed for < 3s) or the factory reset (if pressed for >6s).